.github/ISSUE_TEMPLATE/netbeans_bug_report.yml

```diff
@@ -103,23 +110,30 @@ body:
         Any relevant logs to include? Put them here inside fenced
         ``` ``` blocks or inside a foldable details tag if it's long:
         <details><summary>x.log</summary> lots of stuff </details>
-  - type: checkboxes
+        
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+  - type: dropdown
     attributes:
       label: Are you willing to submit a pull request?
       description: >
         This is absolutely not required, but we are happy to guide you in the contribution process,
-        especially if you already have a good understanding of how to implement the fix.
+        especially if you already have a good understanding of how to implement the fix. <br>
         Apache NetBeans is a community-managed project and we love to bring new contributors in.
+        
       options:
-        - label: Yes I am willing to submit a PR!
-  - type: checkboxes
+        - "Yes"
+        - "No"
+  - type: dropdown
     attributes:
       label: Code of Conduct
-      description: The Code of Conduct helps create a safe space for everyone.
+      description: > 
+         The Code of Conduct helps create a safe space for everyone.
+         I agree to follow the Apache Software Foundation's
+         [Code of Conduct](https://www.apache.org/foundation/policies/conduct.html)
       options:
-        - label: >
-            I agree to follow the Apache Software Foundation's
-            [Code of Conduct](https://www.apache.org/foundation/policies/conduct.html)
+        - "Yes"
+        - "No"
```

Comment:
Sorry, can we lose the `No` option?  I'm assuming dropdowns work fine with only one option, even when required.